applications/crossbar/src/crossbar_services.erl

```diff
@@ -120,24 +120,21 @@ extract_item_from_category(_, ItemJObj, Acc) ->
                           ,wh_transaction:transactions()
                           ,integer()) -> wh_transaction:transactions().
 create_transactions(Context, Item, Acc) ->
-    Quantity = wh_json:get_integer_value(<<"quantity">>, Item, 0),
+    Quantity = wh_json:get_value(<<"quantity">>, Item),
```

Comment:
I think you need to rebase your branch or something. I don't think this is a good change, nor the one below for getting `activiation_charges`